### DISCUSSION TITLE
Feature request: mark a threaded reply as answer to Discussion

### DISCUSSION THREAD

**Discussion by erikr** [community/community#3239] [Discussions] [answered]
In Discussions, users can write Answers, and Replies can be written to these Answers.

Currently, only Answers can be marked as "the answer" (e.g. green check mark):
![Screen Shot 2021-01-26 at 7 40 19 PM](https://user-images.githubusercontent.com/1253422/105925536-53670080-600e-11eb-982b-d394349b4fd1.png)

Comments (replies?) to Answers cannot be marked as "the answer". This would be a good feature to have, because often the answer is in these types of comments:
![Screen Shot 2021-01-26 at 7 37 47 PM](https://user-images.githubusercontent.com/1253422/105925355-f9663b00-600d-11eb-9d74-a429866c879e.png)

Demo Discussion from which above screenshots were taken: https://github.com/aguirre-lab/ml4c3/discussions/1237.

Note this is true even if the Discussion is in a `Q&A` category, and even I am the repo owner, and/or author of the Discussion post. Linking to @becca's comment which may be relevant: https://github.com/github/feedback/discussions/2903#discussioncomment-181405.

> Side comment: Overlapping terminology makes this explanation a little confusing ("Answers", "Comments", "Replies", "Discussions"). Perhaps precise and distinct terms would help.

**Comment by becca** [comment]
Currently, only top-level/parent comments can be marked as an answer. 

This is a feature request we're tracking and we'll do our best to keep you posted on its priority within our roadmap. cc// @evi-liu

**Reply by jay** [reply]
Do you have an update on this?

**Reply by jricciardi** [reply, marked as answer]
Hey @erikr and @jay! Good news, this feature has now shipped 🚀  [Check out the change log for more details](https://github.blog/changelog/2023-04-17-marking-a-threaded-comment-as-the-answer/) (and the fact this threaded comment is marked as the answer 😄 )

**Reply by erikr** [reply]
Nice work, congrats on shipping! This will be useful.

**Comment by usman-302** [comment]
how can I open a issue

**Comment by usman-302** [comment]
how can I open a issue
